resources/views/people/documents/index.blade.php

```diff
@@ -0,0 +1,5 @@
+<div class="{{ \App\Helpers\LocaleHelper::getDirection() == 'ltr' ? 'fl' : 'fr' }} w-100 pb3 pt1 pl3 pr3">
```

Comment:
Use the helper `htmldir()`
```suggestion
<div class="{{ htmldir() == 'ltr' ? 'fl' : 'fr' }} w-100 pb3 pt1 pl3 pr3">
```